### PR TITLE
Ember Best Practices: How to avoid leaking state into factories

### PR DESCRIPTION
**Pull request by brzpegasus**

### CODE REVIEW

File: source/posts/2015-09-18-back-to-the-basics-with-emberobject.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+twitter: edeblois
+github: brzpegasus
+author: "Estelle DeBlois"
+tags: ember, javascript
+social: true
+comments: true
+published: true
+---
+
+`Ember.Object` is one of the first things we learn as Ember developers, and to no surprise. Pretty much every object we work with in Ember, whether it's a route, component, model, or service, extends from `Ember.Object`. But every now and then, I see it used incorrectly, like this:
```
Comment (bcardarella) [inline source/posts/2015-09-18-back-to-the-basics-with-emberobject.md]: To help drive this home you could link to my Ember.Object graph: http://emberjs.jsbin.com/boqapo

Comment (brzpegasus) [inline source/posts/2015-09-18-back-to-the-basics-with-emberobject.md]: :+1:

File: source/posts/2015-09-18-back-to-the-basics-with-emberobject.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+});
+```
+
+New objects of type `Post` can now be created by calling `Post.create()`. Each post instance will inherit the properties and methods that have been defined on the `Post` class:
+
+```js
+const post = Post.create();
+post.get('title'); // => 'Untitled'
+post.get('author'); // => 'Unknown'
+post.get('header'); // => 'Untitled by Unknown'
```
Comment (bcardarella) [inline source/posts/2015-09-18-back-to-the-basics-with-emberobject.md]: might just be me but the first time I read this I was confused as I thought somehow something was happening with `undefined`

Comment (brzpegasus) [inline source/posts/2015-09-18-back-to-the-basics-with-emberobject.md]: lol I can replace `Unknown` with `Anonymous`.

**Comment by brzpegasus** [comment]
Updated per feedback. I changed the title and added a paragraph to introduce readers to the series.

**Comment by bcardarella** [comment]
:+1: looks good to me. Anybody else have any blockers before merging?

**Comment by poteto** [comment]
LGTM

**Pull request merged**